test/stability/testCksum/checksum_test.go

```diff
@@ -8,9 +8,36 @@ import (
 	"testing"
 )
 
+
+func TestInit(t *testing.T) {
+	if err := initTest(); err != nil {
+        t.Fatalf("fail to initialize with error: %+v\n", err)
+    }
+}
+// TODO unfortunately this test can't be executed as go test right now due to https://github.com/intel-go/nff-go/issues/301
+// If you want to test it via go test you should firstly comment FreeMempools function at flow/flow.go
+// You will have multiple mempools however test will pass.
 func TestChecksum(t *testing.T) {
-	if err := executeTest(0); err != nil {
-		t.Logf("fail: %+v\n", err)
-		t.Fail()
+	variants := []bool {true, false}
+	for _, useIPv4 := range variants {
+		for _, useIPv6 := range variants {
+			for _, useUDP := range variants {
+				for _, useTCP := range variants {
+					for _, useICMP := range variants {
+						if useICMP && (useTCP || useUDP) {
+							continue
+						}
+						for _, useVLAN := range variants {
+							err := executeTest(gotest, useIPv4, useIPv6, useUDP, useTCP, useICMP, useVLAN)
+							resetState()
+							if err != nil {
+								t.Logf("useIPv4=%t, useIPv6=%t, useUDP=%t, useTCP=%t, useICMP=%t, useVLAN=%t configuration failed with: %+v\n", useIPv4, useIPv6, useUDP, useTCP, useICMP, useVLAN, err)
+								t.Fail()
```

Comment:
or failTest()?

Reply:
no, these functions are gotest specific, they should be used.